services/jitpack/jitpack-downloads.tester.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const t = (module.exports = require('../tester').createServiceTester())
+
+const isMetric = /^.*\/(week|month)/
```

Comment:
I'd suggest using the existing `isMetricOverTimePeriod` validator from `test-validators` instead. I think this would technically pass on `abc/month` which is not ideal 😄

Reply:
I must have seen `isMetric` somewhere in the codebase (probably in pypi service` and have tried to imitate that. Whatever, I will use `isMetricOverTimePeriod`. :+1: